ml-metadata/rockcraft.yaml

```
@@ -0,0 +1,79 @@
+# Based on: https://github.com/google/ml-metadata/blob/v1.14.0/ml_metadata/tools/docker_server/Dockerfile
+name: ml-metadata
+summary: Part of the ML Metadata (MLMD) framework.
+description: |
+  ML Metadata is a framework for managing and tracking metadata associated with ML workflows. 
+  It is part of the TensorFlow Extended (TFX) ecosystem, but it can be used independently as well.
+version: "1.14.0"
+license: Apache-2.0
+# Build does not work with ubuntu@22.04. Issue: https://github.com/google/ml-metadata/issues/210
+base: ubuntu@20.04
+run-user: _daemon_
+platforms:
+  amd64:
+
+services:
+  mlmd:
+    override: replace
+    summary: "ml metadata store service"
+    startup: enabled
+    environment:
+      GRPC_PORT: "8080"
+      METADATA_STORE_SERVER_CONFIG_FILE: ""
+    command: "/bin/metadata_store_serve --grpc_port=${GRPC_PORT} --metadata_store_server_config_file=${METADATA_STORE_SERVER_CONFIG_FILE}"
```

there's a typo with the binary in the command
```suggestion
    command: "/bin/metadata_store_server --grpc_port=${GRPC_PORT} --metadata_store_server_config_file=${METADATA_STORE_SERVER_CONFIG_FILE}"
```